src/jitter.rs

```diff
@@ -115,6 +115,13 @@ impl ::std::error::Error for TimerError {
     }
 }
 
+impl From<TimerError> for Error {
```

Comment:
Became necessary because of `NewSeeded`.

Reply:
Looks like something I forgot.